src/Protocol/ActivityPub/Transmitter.php

```diff
@@ -882,7 +882,7 @@ private static function isAPContact(array $contact, array $networks): bool
 			return true;
 		}
 
-		return GServer::getProtocol($contact['gsid'] ?? 0) == Post\DeliveryData::ACTIVITYPUB;
+		return ($contact['gsid'] > 0 ? GServer::getProtocol($contact['gsid']) : null) === Post\DeliveryData::ACTIVITYPUB;
```

Comment:
It should look like this after you're done:
```suggestion
		return GServer::getProtocol($contact['gsid'] ?? 0) === Post\DeliveryData::ACTIVITYPUB;
```

Reply:
In the original (not my modified) version `GServer::getProtocol()` returns `null` but here (and below) it is "translated" to `0`. Let's return `0` and avoid this extra null-coalescing.

Reply:
I'm not sure what you're talking about, the null coalescing is for the `$contact['gsid']` variable, not the return value.